scripts/import/uniprot/uniprot_importer.py

```diff
@@ -46,11 +46,22 @@ def get_batch(batch_url):
         yield response
         batch_url = get_next_link(response.headers)
 
-def get_database_value(database, dataItem):
+def get_database_value(database, dataItem, gene_symbol):
     if "uniProtKBCrossReferences" in dataItem and len(dataItem["uniProtKBCrossReferences"]):
         for item in dataItem["uniProtKBCrossReferences"]:
-            if item["database"] == database:
+
+            # Example: {'database': 'HGNC', 'id': 'HGNC:4764', 'properties': [{'key': 'GeneName', 'value': 'H3-3A'}]
+            if database == "HGNC" and item["properties"][0]["value"] == gene_symbol:
+                return item["id"]
+
+            # Example: {'database': 'MIM', 'id': '601058', 'properties': [{'key': 'Type', 'value': 'gene'}]}
+            elif database == "MIM" and item["properties"][0]["value"] == "gene":
                 return item["id"]
+
+            # This checks if there is a HGNC ID
+            elif gene_symbol is None:
```

Comment:
better to have the additional condition -> item["database"] == database

Reply:
Yes, here it could also include `item["database"] == database`